src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -67,16 +68,105 @@ const colors = {
   get borderDark() {
     return colors.gray400;
   },
+} as const;
 
-  borderRadius: '4px',
-  borderRadiusBottom: '0 0 4px 4px',
-  borderRadiusTop: '4px 4px 0 0',
-  headerSelectorRowHeight: 44,
-  headerSelectorLabelHeight: 28,
-
-  dropShadowLightest: '0 1px 2px rgba(0, 0, 0, 0.04)',
-  dropShadowLight: '0 2px 0 rgba(37, 11, 54, 0.04)',
-  dropShadowHeavy: '0 1px 4px 1px rgba(47,40,55,0.08), 0 4px 16px 0 rgba(47,40,55,0.12)',
+const aliases = {
+  /**
+   * Primary text color
+   */
+  textColor: colors.gray800, // TODO(dark): colors.gray500
+
+  /**
+   * Text that should not have as much emphasis
+   */
+  subText: colors.gray400,
+
+  /**
+   * A color that denotes a "success", or something good
+   */
+  success: colors.green400, // TODO(dark): colors.green300,
+
+  /**
+   * A color that denotes an error, or something that is wrong
+   */
+  error: colors.red400, // TODO(dark): colors.300,
+
+  /**
+   * Primary border color
+   */
+  border: colors.gray200,
```

Comment:
Do lighter/darker borders need aliases as well?

Reply:
The intent (atm) will be to only use a singular border color

Reply:
🙏